File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
       int[] arrList = new int[nums.length];
       int left = 1;
       int right =1;
       for(int i=0; i<nums.length; i++){
        arrList[i]=left;
        left=left*nums[i];
       }
       for(int j=(nums.length)-1; j>=0; j--){
        arrList[j]=arrList[j]*right;
        right = right*nums[j];
       }
       return arrList;
    }
    
}  

File: Data Structures & Algorithms/kth-smallest-integer-in-bst/submission-1.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def kthSmallest(self, root: Optional[TreeNode], k: int) -> int:
        curr =[]
        def bfs(root):
            if not root:
                return 0
            bfs(root.left)
            curr.append(root.val)
            bfs(root.right)

        bfs(root)
        return curr[k-1]
            

